Recrutify/Recrutify.Host/Configuration/Profiles/ProjectProfile.cs

```diff
@@ -13,12 +13,14 @@ public ProjectProfile()
         {
             CreateMap<CreateProjectDTO, Project>()
```

Comment:
Make mapping from BaseProjectDTO to Project and include all common mappings from Create and Update not to duplicate, same for other similar situations - Candidate, or smth else
use IncludeAllDerived

Reply:
https://docs.automapper.org/en/stable/Mapping-inheritance.html

Reply:
done